tests/bot/cogs/moderation/test_modlog.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+import discord
+
+from bot.cogs.moderation.modlog import ModLog
+from tests.helpers import MockBot, MockTextChannel
+
+
+class ModLogTests(unittest.IsolatedAsyncioTestCase):
+    """Tests for moderation logs."""
+
+    def setUp(self):
+        self.bot = MockBot()
+        self.cog = ModLog(self.bot)
+        self.channel = MockTextChannel()
+
+    async def test_log_entry_description_truncation(self):
+        """Should truncate embed description for ModLog entry."""
```

Comment:
Should be written in imperial mood.

```suggestion
        """Test that embed description for ModLog entry is truncated."""
```